ca_ab_edmonton/people.py

```diff
@@ -9,22 +9,24 @@ class EdmontonPersonScraper(CanadianScraper):
 
     def scrape(self):
         yield self.scrape_mayor()
-
         page = self.lxmlize(COUNCIL_PAGE)
-        councillors = page.xpath('//th[contains(text(), "Ward")]')
+
+        councillors = page.xpath('//div[contains(@class, "documentexcerpt-module__item")]')
         assert len(councillors), 'No councillors found'
         for cell in councillors:
-            district = cell.text
             name = cell[1].text
             if name != 'Vacant':
-                page_url = cell[1].attrib['href']
+                page_url = cell[0].attrib['href']
                 page = self.lxmlize(page_url)
+                district_name = page.xpath('//h1[contains(@class, "page-title")]')[0].text_content()
+                district = district_name.split(' - ')[0]
```

Comment:
You can assign name and district in one line with:

    district, name = district_name.split(' - ', 1)